Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/AssetGroups/ByAssetGroup.tsx

```diff
@@ -206,21 +214,12 @@ const ByAssetGroup: Application.Types.iByComponent = (props) => {
 
     function getEnum(setOptions, field) {
         let handle = null;
-        if (field.type != 'enum' || field.enum == undefined || field.enum.length != 1)
+        if (field.type != 'enum')
             return () => { };
 
-        handle = $.ajax({
-            type: "GET",
-            url: `${homePath}api/ValueList/Group/${field.enum[0].Value}`,
-            contentType: "application/json; charset=utf-8",
-            dataType: 'json',
-            cache: true,
-            async: true
-        });
-
-        handle.done(d => setOptions(d.map(item => ({ Value: item.Value.toString(), Label: item.Text }))))
-        return () => {
-            if (handle != null && handle.abort == null) handle.abort();
```

Comment:
Don't we need this code to get enum values from the `ValueList` table for fields defined in the `AdditionalField` table?

Reply:
I'm pretty sure that's wrong. The records in the `AdditionalField` table are brought in by the `getAdditionalMeterFields()` and `getAdditionalAssetFields()` functions. If their `Type` column is set to enum, then this code would get a reference to `ValueListGroup.ID` from the `AdditionalFieldValue` table for that field and look up the enum values from there.

Reply:
I can assure that its not. Also, the previous code no longer works in general, which was what was causing issues. I don't think enum is ever defined by the calling function in enumsetter within SearchBar. If we want to use this old code, we need to search the valuelist table via a different property I think. field.key should still work for that, thoughts?